swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        unique_identifier = uuid.uuid4()
+        self.id = id if id else unique_identifier.int
+        if not isinstance(self.id, int):
+            raise TypeError("Item id should be an int type object")
+        self.condition = condition if condition else 0
+
+    def get_category(self):
+        return self.__class__.__name__
+        
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}"
+
+    def condition_description(self):
```

Comment:
Well organized code! At this point, you might see that a `dict` could be an alternative.  And, has O(1) lookup time.  In the worse case, it would take O(n) -- for n conditions -- to fall through to the last condition case with the `if/else` structure. :)